src/sentry/integrations/bitbucket/repository.py

```diff
@@ -59,6 +59,7 @@ def validate_config(self, organization, config):
                 self.raise_error(e, identity=client.auth)
             else:
                 config['external_id'] = six.text_type(repo['uuid'])
+                config['integration_id'] = six.text_type(repo['integration_id'])
```

Comment:
So the integration_id you want should be the same as`config['integration_id']` and it shouldn't be a string: https://github.com/getsentry/sentry/blob/master/src/sentry/models/repository.py#L27